docs/blog/.authors.yml

```diff
@@ -3,3 +3,7 @@ authors:
     name: Sid
     description: Data Science Director
     avatar: https://raw.githubusercontent.com/IDinsight/aaq-core/main/docs/blog/author-avatars/sid.png
+  Carlos:
+    name: Carlos
+    description: Data Scientist
+  
```

Comment:
Suggest adding your image as well. See where mine is and the path. It will only work once merged in to main.